success-criteria.md

```diff
@@ -237,9 +222,7 @@ these supply-chain vendors is for cross-publisher budget management services,
 independent verification, and authentication services, their objectivity is
 necessary for accountability, transparency and reconciliation purposes. This
 interoperability is a goal in support of W3C's mission and the first principle
-in support of improved web advertising. The trust in this interoperable network
-requires transparency and improved documentation of acceptable and unacceptable
-uses of data. Organizations must be transparent about the personal information
+in support of improved web advertising. The trust in this interoperable network requires transparency, reliable and consistent controls for both end users and publishers, and improved documentation of acceptable and unacceptable uses of data. Organizations must be transparent about the personal information
```

Comment:
Interoperable networks in general and this one in specific does not just require improved documentation but also improved and reliable controls. As an example: publishers may want to set specific higher levels of privacy beyond defaults while still using vendors that can serve more than one site. Proposed rephrase.

Reply:
I think it's important to think beyond transparency & control. Those defined ethical data processing in the 70s, but things have changed a bit since.